src/commands/spotify.js

```diff
@@ -0,0 +1,23 @@
+import { PERSONALITY } from "./personality.js";
+import { parseLink, spotifyReply } from "../helpers/index.js";
+
+const action = async (message, _personality, client, currentServer) => {
+
+  const foundLink = await parseLink(
+    message.content,
+    client,
+    PERSONALITY.spotify,
+    currentServer
+  );
+
+  await spotifyReply(foundLink, message, client, currentServer);
+};
+
```

Comment:
Decide if the spotifyHelper.js file is really usefull or should be merged with spotify.js file.